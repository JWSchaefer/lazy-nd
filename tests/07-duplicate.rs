use lazy_nd::lazy_nd;
#[lazy_nd(dim = D)]
struct Test<const D: usize = 2, T> {
    #[scalar(mass:f64)]
    #[scalar(position : f64)]
    #[scalar(position : f32)]
    field: f64,
}
fn main() {}
