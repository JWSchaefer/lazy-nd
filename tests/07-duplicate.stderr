error: field `position` already declared
 --> tests/07-duplicate.rs:6:14
  |
6 |     #[scalar(position : f32)]
  |              ^^^^^^^^
